pkg/discovery/discovery.go

```diff
@@ -80,6 +81,17 @@ func Run(kubeClient kubernetes.Interface, cfg *config.Config) []error {
 		errlst = append(errlst, err)
 	}
 
+	// 8. Always write the done file
+	doneFile := cfg.ResultsDir + "/done"
```

Comment:
Hmm I'd almost prefer to have this in the script wrapper, b/c it makes the contract super light.  

What benefits are there to adding this to code vs. script wrapper?